addons/options/README.md

```diff
@@ -1,136 +1,28 @@
-#NOTE: Options Addon is deprecated as of Storybook 5.0
+# Options Addon is deprecated as of Storybook 5.0
 
-Options are now configured using the [`options` parameter](../../docs/src/pages/configurations/options-parameter/index.md) which is built into Storybook.
-
-- Global options: `addParameters({ options: { ... }})` and no addon is needed.
-- Story options: `storiesOf(...).add('name', storyFn, { options: { ... }})`
-
-See the [migration docs](../../MIGRATION.md#options-addon-deprecated) for what's changed.
-
-# Storybook Options Addon
-
-The Options addon can be used to (re-)configure the [Storybook](https://storybook.js.org) UI at runtime.
-
-[Framework Support](https://github.com/storybookjs/storybook/blob/master/ADDONS_SUPPORT.md)
-
-![Screenshot](https://raw.githubusercontent.com/storybookjs/storybook/HEAD/addons/options/docs/screenshot.png)
-
-## Getting Started
-
-First, install the addon
-
-```sh
-yarn add @storybook/addon-options --dev
-```
-
-Add this line to your `addons.js` file (create this file inside your storybook config directory if needed).
-
-```js
-import '@storybook/addon-options/register';
-```
-
-### Set options globally
-
-Import and use the `addParameters` + `options`-key in your `config.js` file.
+Global options can be specified with:
 
 ```js
-import { addParameters, configure } from '@storybook/react';
-
-// Option defaults:
-addParameters({
```

Comment:
We don't want to keep these JSdocs in this file (for the types of options?)

Reply:
No, this was duplicated, I just refer to the docs site now.